lib/commandlineio.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "blackjack_score"
+
```

Comment:
If I use just 'require' I get an error saying the file can't be loaded. Maybe my code is wrong? I definitely need to research this though.